nix/shared.nix

```diff
@@ -236,7 +236,7 @@ let
                   [ (pkgsNew.haskell.lib.packagesFromDirectory { directory = ./packages; })
                     extension
                     doCheckExtension
-                    doBenchmarkExtension
+                    # doBenchmarkExtension
```

Comment:
Should this be a separate PR?  Wasn't sure if you'd meant to include this change.

Reply:
Yeah, that's a mistaken change that I accidentally committed.  I'll remove that